.cbmc-batch/jobs/aws_cryptosdk_enc_ctx_serialize/aws_cryptosdk_enc_ctx_serialize_harness.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <aws/common/hash_table.h>
+#include <aws/common/private/hash_table_impl.h>
+#include <aws/cryptosdk/enc_ctx.h>
+#include <proof_helpers/make_common_data_structures.h>
+#include <proof_helpers/proof_allocators.h>
+#include <proof_helpers/utils.h>
+
+void array_list_item_generator(struct aws_array_list *elems) {
```

Comment:
This looks like the kind of allocator you usually extract to a file of generic utility generators.  Is this one unusual?

Reply:
I believe this is being used as a generator function as described in the comment in aws_cryptosdk_hash_elems_array_init_stub.c. Also see line 36 of the Makefile.

Suggest writing a comment---perhaps just a brief one, pointing to aws_cryptosdk_hash_elems_array_init_stub.c for a full explanation.

Reply:
Done